cypress/commons/actions/generic/DatasetManager.js

```diff
@@ -15,8 +15,81 @@ function switchToDatasetManagerView() {
   return getDatasetManagerTab().click();
 }
 
+function getNoDatasetsPlaceholder() {
```

Comment:
Should not _dataset_ be in the singular if it has none ?

Reply:
In English, "no", "zero" and "any" are followed by plural nouns :slightly_smiling_face: